core/src/main/scala/dagr/core/cmdline/parsing/ClpReflectiveBuilder.scala

```diff
@@ -211,32 +206,13 @@ private[parsing] class ClpArgument(declaringClass: Class[_],
       case _ => throw new IllegalArgumentException(s"Could not determine collection type: ${input.getClass.getCanonicalName}")
     }
 
-    def size: Int = {
-      (seq.isDefined, set.isDefined) match {
-        case (true, _) => seq.get.size
-        case (_, true) => set.get.size
-        case _ => collection.get.size
-      }
-    }
+    def size: Int = (seq orElse set) map (_.size) getOrElse collection.get.size()
 
-    def isEmpty: Boolean = {
-      (seq.isDefined, set.isDefined) match {
-        case (true, _) => seq.get.isEmpty
-        case (_, true) => set.get.isEmpty
-        case _ => collection.get.isEmpty
-      }
-    }
+    def isEmpty: Boolean = size == 0
```

Comment:
I will change this to the style of `size` since some underlying implementations may not need to get an accurate count to determine emptiness.